docs/_data/table/en/user-configuration.yml

```diff
@@ -33,6 +33,8 @@ alluxio.user.file.writetype.default:
   Valid options are `MUST_CACHE` (write will only go to Alluxio and must be stored in Alluxio),
   `CACHE_THROUGH` (try to cache, write to UnderFS synchronously), `THROUGH` (no cache, write to
   UnderFS synchronously).
+alluxio.user.file.cache.incomplete.block:
+  A block is cached to Alluxio even if the block is not fully read when this flag is enabled.
```

Comment:
NIT (try to be more explicit):

When read type is `CACHE_PROMOTE` or `CACHE` and this property is set to true, the entire block will be cached by Alluxio space even if the client only reads a part of this block.